src/firejail/fs_trace.c

```diff
@@ -41,6 +41,27 @@ void fs_trace_preload(void) {
 		fclose(fp);
 		fs_logger("touch /etc/ld.so.preload");
 	}
+	if (arg_tracefile) {
```

Comment:
should we break this out into an own function? fs_trace_preload() is called up to three times, whereas this code only needs to run once.